packages/framework/src/Framework/Factories/NavigationDataFactory.php

```diff
@@ -210,6 +210,7 @@ private function canUseSubdirectoryForGroups(): bool
 
     private function defaultGroup(): ?string
     {
+        // TODO: It would be better if this was null in all cases, considering 'other' is used a a sort of confusing pseudo-null
```

Comment:
It would be better if this was null in all cases, considering 'other' is used a a sort of confusing pseudo-nul

Reply:
https://github.com/hydephp/develop/pull/1572